selenium_tests/base.py

```diff
@@ -1,4 +1,9 @@
-"""Basic selenium tests for MicroMasters"""
+"""
+NOTE: THIS MODULE IS DEPRECATED AND SHOULD BE REMOVED ONCE THE 'snapshot_dashboard_states' COMMAND IS
```

Comment:
I'm a bit uneasy about leaving this here. What's left to do to make `snapshot_dashboard_states` work?